core/templates/components/summary-tile/exploration-summary-tile.component.spec.ts

```diff
@@ -223,6 +223,12 @@ describe('Exploration Summary Tile Component', () => {
     fixture.detectChanges();
   });
 
+  it('should get RTL language status correctly', () => {
+    spyOn(
+      i18nLanguageCodeService, 'isCurrentLanguageRTL').and.returnValue(true);
+    expect(component.isLanguageRTL()).toBeTrue();
```

Comment:
```suggestion
    spyOn(
      i18nLanguageCodeService, 'isCurrentLanguageRTL').and.returnValue(true);
 
    expect(component.isLanguageRTL()).toBeTrue();
```

Reply:
Please change the value and test the func.
Follow a similar format as given below:

//Pre-check
//Action
//post-check

Please do this for all the tests.

Reply:
Done

Reply:
@aks681 I don't see the changes.

Reply:
This component doesn't require this function anymore, removed.